include/variables.h

```diff
@@ -2679,31 +2679,31 @@ extern volatile OSTime sRSPOtherTotalTime;
 // extern UNK_TYPE1 D_801FBB18;
 extern volatile OSTime gRDPTotalTime;
 // extern UNK_TYPE1 D_801FBB28;
-extern OSViMode D_801FBB30;
-extern u32* gFramebuffers[2];
-extern OSViMode* D_801FBB88;
+extern OSViMode sNotebookViMode;
+extern void* gFramebuffers[2];
+extern OSViMode* gActiveViMode;
 extern u16* gZBufferPtr;
-extern void* D_801FBB90;
+extern void* gWorkBuffer;
 extern u64* gGfxSPTaskOutputBufferPtr;
-extern size_t gGfxSPTaskOutputBufferSize;
-// extern UNK_TYPE1 D_801FBB9C;
-// extern UNK_TYPE1 D_801FBBA0;
-extern u16 (*gZBuffer)[SCREEN_WIDTH * SCREEN_HEIGHT];
-extern u16 (*gWorkBuffer)[SCREEN_WIDTH * SCREEN_HEIGHT];
-extern u64 (*gGfxSPTaskOutputBuffer)[0x3000];
-extern void* gGfxSPTaskOutputBufferEnd;
-// extern UNK_TYPE1 D_801FBBB4;
-// extern UNK_TYPE1 D_801FBBB8;
-// extern UNK_TYPE1 D_801FBBBC;
-// extern UNK_TYPE1 D_801FBBC0;
-extern u64 (*gGfxSPTaskOutputBuffer2)[0x3000];
-extern void* gGfxSPTaskOutputBufferEnd2;
-extern UNK_TYPE2 D_801FBBCC;
-extern UNK_TYPE2 D_801FBBCE;
-// extern UNK_TYPE1 D_801FBBD0;
-// extern UNK_TYPE1 D_801FBBD2;
-// extern UNK_TYPE1 D_801FBBD4;
-// extern UNK_TYPE1 D_801FBBE0;
+extern void* gGfxSPTaskOutputBufferSize;
+extern void* sCfbLoRes1;
+extern void* sCfbLoRes0;
+extern u16 (*gZBufferLoRes)[SCREEN_WIDTH * SCREEN_HEIGHT];
+extern u16 (*gWorkBufferLoRes)[SCREEN_WIDTH * SCREEN_HEIGHT];
+extern u64 (*gGfxSPTaskOutputBufferLoRes)[0x3000];
+extern void* gGfxSPTaskOutputBufferSizeLoRes;
+extern void* sCfbHiRes1;
+extern void* sCfbHiRes0;
```

Comment:
I think some of these externs are local. Could you remove the externs that are local to minimize changes to bss?

Reply:
Removing the externs results in non-matching, as doing so affects the .bss order of sys_cfb's variables.

Reply:
You should be able to use the bss scripts to resolve it?

Reply:
I don't really see how, since it's sys_cfb's own variables that are being shifted

Reply:
Adding `#include "prevent_bss_reordering.h"` to the sys_cfb file does not fix the bss issue? :thinking:

Reply:
The main issue is bss gets ordered based on externing as well, so if you don't extern everything things get ordered differently. Same issue in play, where everything had to be externed even locals.

Reply:
Dang, tricky. That's fine then